src/components/WebexMeetingGuestAuthentication/WebexMeetingGuestAuthentication.jsx

```diff
@@ -58,11 +65,26 @@ export default function WebexMeetingGuestAuthentication({
     phone: width <= PHONE_LARGE,
   });
 
+  let passwordError = '';
+  let captchaError = '';
+
+  if (invalidPassword && failureReason === 'WRONG_PASSWORD') {
+    if (!requiredCaptcha.captchaId) {
+      passwordError = 'Incorrect password. Try again.';
+    } else {
+      captchaError = 'Incorrect password entered too many times. Enter captcha code';
+    }
+  } else if (failureReason === 'WRONG_CAPTCHA' && requiredCaptcha.captchaId) {
+    captchaError = 'Invalid Captcha. Try again.';
+  }
+
   const isStartButtonDisabled = nameError || !password || invalidPassword || isJoining;
 
   const joinMeeting = () => {
     setIsJoining(true);
-    adapter.meetingsAdapter.joinMeeting(ID, {name, password}).finally(() => setIsJoining(false));
+    adapter.meetingsAdapter
+      .joinMeeting(ID, {name, password, captcha})
+      .finally(() => setIsJoining(false));
```

Comment:
What happens if joinMeeting fails?

Reply:
Failure scenario is handled in the sdk-component-adapter. Password error and captcha error are both failure scenarios, if we encounter that, then in the adapter we update the meetings object and that change is then reflected here.